src/code/z_message.c

```diff
@@ -5217,14 +5217,14 @@ s16 sTextboxMidYPositions[] = {
 
 s16 D_801D0448[] = { 0x1C, 0x1D, 0x1E, 0x1F, 0x20, 0x21, 0x22, 0x23, 0x24, 0x25 };
 
-s16 D_801D045C[] = {
-    0x1B91,
-    0x1B90,
-    0x1B8F,
+s16 sDoubleTimeSkipToNightMessageIDs[] = {
```

Comment:
```suggestion
s16 sDoubleTimeSkipToNightTextIds[] = {
```
Pretty sure we use textId more often.